src/MacroTools.jl

```diff
@@ -14,10 +16,7 @@ include("examples/destruct.jl")
 include("examples/threading.jl")
 include("examples/forward.jl")
 
-function __init__()
-  animals_file = joinpath(dirname(@__FILE__), "..", "animals.txt")
-  global const animals =
-    shuffle(Symbol.(lowercase.(split(read(animals_file, String)))))
-end
+animals_file = joinpath(dirname(@__FILE__), "..", "animals.txt")
+const animals = shuffle(Symbol.(lowercase.(split(read(animals_file, String)))))
```

Comment:
Julia v0.7 doesn't allow `global const` inside functions, so I've just removed `__init__` entirely. This should be perfectly safe to do, even with precompilation. The only difference is that now the order of animals doesn't change each time the module is re-loaded (it's just shuffled at compile-time). If you want that behavior back, I do that.

Reply:
I think just using `global` would be fine, it doesn't need to be `const`.

Reply:
Ok, I've put it back.